zeppelin-zengine/src/test/java/org/apache/zeppelin/notebook/NotebookTest.java

```diff
@@ -470,6 +471,29 @@ public void testRemoveNote() throws IOException, InterruptedException {
     }
   }
 
+  @Test
+  public void testRemoveCorruptedNote() throws IOException, InterruptedException {
+    try {
+      LOGGER.info("--------------- Test testRemoveCorruptedNote ---------------");
+      // create a note and a paragraph
+      Note corruptedNote = notebook.createNote("note1", anonymous);
+      String corruptedNotePath = notebookDir.getAbsolutePath() + corruptedNote.getPath() + "_" + corruptedNote.getId() + ".zpln";
+      // corrupt note
+      try (FileWriter myWriter = new FileWriter(corruptedNotePath)) {
+        myWriter.write("{{{I'm corrupted;;;");
+      } catch (IOException e) {
```

Comment:
Same, no need to catch the `IOException`.

Reply:
Done